src/System.Text.Json/src/System/Text/Json/Node/JsonObject.cs

```diff
@@ -378,8 +164,9 @@ public void Add(string propertyName, IEnumerable<JsonNode> propertyValues)
             var jsonArray = new JsonArray();
```

Comment:
I guess the purpose of this method is so you can do something like `obj.Add("numbers", arr.OfType<JsonNumber>())`?

Is that common enough, useful enough, and discoverable enough, that we want the method here instead of just getting the user to wrap their enumerable in `new JsonArray(...)`?

Reply:
Yes, it's just for convenience. I can remove it for now.